### PR TITLE
Cit-iftyed Box map as well as adding back blasters back to armory

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Corrects a few armory having 4 tasers, or missing a shotgun for some reason...
Adds 2 blasters for every armory
Adds back in Box's clown/mime room
Moves the Nuke on Box to the middle of the map
Moves the Grav gene to engi department.

## Why It's Good For The Game

These changes with box station were unintented to be removed as it turns it out
The removal of blaster shotguns were unintented.

## Changelog
:cl:
add: Blaster shotguns back into armory
/:cl:

### CODE REVIEW

**Comment by GrayRachnid** [comment]
Brig entrance too small, why are you removing the 4th cell?

**Comment by Trilbyspaceclone** [comment]
@GrayRachnid The old cit box map didnt even have a second door...
Im trying to make it to how it was back in the day to be better*

File: code/datums/ruins/station.dm [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+	name = "Engine SM"
+
+/datum/map_template/ruin/station/box/engine/singulo
+	id = "engine_singulo"
+	suffix = "Box/Engine/engine_singulo.dmm"
+	name = "Engine Singulo"
+
+/datum/map_template/ruin/station/box/engine/tesla
+	id = "engine_tesla"
+	suffix = "Box/Engine/engine_tesla.dmm"
```
Comment (Ghommie) [inline code/datums/ruins/station.dm:20]: whitespace changes, most likely line endings. :pouting_man:

**Review by Ghommie — APPROVED** [review]
I'm not a fan of that brig change, please revert it back. Keep the water cooler around, but the rest can go.
The rest is gucci more or less, yet the +/- count is a tad too high. Also glad to see the mime/clown got their own theatre room back.

File: code/datums/ruins/station.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/datums/ruins/station.dm:20]: I didnt touch this file idk what happend to it

**Review by Ghommie — CHANGES_REQUESTED** [review]
Move the chairs, water cooler and leftmost table (and its the water cups box) one tile left. Remove the rest and bring back the vendors (from left to right: coffee, random cola and cigs).

**Comment by Trilbyspaceclone** [comment]
@Ghommie No. The chairs, water cooler, venders are all in tactical places.
Tripping hazards as a real problem.
And venders being in a cubby were you also can stand is a reasonable thing

**Review by Ghommie — CHANGES_REQUESTED** [review]
Bring back the vendors and remove the soda dispenser. We already got one at the bar and another in the maint bar. Don't try to change my mind.

**Comment by Trilbyspaceclone** [comment]
@Ghommie Old box cit map had those in that area. 
Im keeping them their

**Review by Ghommie — APPROVED** [review]

**Comment by Ghommie** [comment]
Guess I'll remove the extra soda dispenser in a later PR.

**Comment by KathrinBailey** [comment]
Upload some pics of all the changes.

**Comment by IronEleven** [comment]
Did stealing from the vault really need to be this much harder?

**Comment by Ghommie** [comment]
Unintended side-effect probably, yikes.

**Pull request merged**